src/reducers/port.reducer.test.js

```diff
@@ -27,61 +27,57 @@ describe('Check port reducer', () => {
 			type: 'FLOWDESIGNER_PORT_ADD',
 			nodeId: 'nodeId',
 			portId: 'portId',
-			portType: 'portType',
-			data: { type: 'EMITTER' },
-			attributes: { clicked: true },
+			graphicalAttributes: { portType: 'portType', properties: { type: 'EMITTER' } },
 		})).toMatchSnapshot();
 	});
 
-	it('FLOWDESIGNER_PORT_ADDS to add multiple ports to port collection', () => {
+	it('FLOWDESIGNER_PORT_ADDS to add multiple ports (portId1, portId2) to port collection', () => {
 		expect(portReducer(initialState, {
 			type: 'FLOWDESIGNER_PORT_ADDS',
 			nodeId: 'nodeId',
 			ports: [{
 				portId: 'portId1',
-				portType: 'portType',
-				data: { type: 'EMITTER' },
+				graphicalAttributes: { portType: 'portType', properties: { type: 'EMITTER' } },
```

Comment:
this is a mocked action where datastructure are kept in simple literral javascript object